sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/Models/BatchPoolIdentity.cs

```diff
@@ -36,10 +36,8 @@ public BatchPoolIdentity()
         /// <param name="type">The type of identity used for the Batch Pool.
         /// Possible values include: 'UserAssigned', 'None'</param>
         /// <param name="userAssignedIdentities">The list of user identities
-        /// associated with the Batch pool. The user identity dictionary key
-        /// references will be ARM resource ids in the form:
-        /// '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.</param>
-        public BatchPoolIdentity(PoolIdentityType type, IDictionary<string, BatchPoolIdentityUserAssignedIdentitiesValue> userAssignedIdentities = default(IDictionary<string, BatchPoolIdentityUserAssignedIdentitiesValue>))
```

Comment:
same comment - replacing the identity type is a breaking change, but since these types appear to be equivalent, we may be able to fix by re-introducing the old type.

Reply:
still a breaking change - was there somne mitigation that you intended?